functions/Export-DbaInstance.ps1

```diff
@@ -173,7 +177,7 @@ function Export-DbaInstance {
             } catch {
                 Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
-            $timeNow = (Get-Date -uformat "%m%d%Y%H%M%S")
+            $timeNow = (Get-Date -UFormat (Get-DbatoolsConfigValue -FullName 'formatting.uformat'))
```

Comment:
Do we want to add a short note to this in the comments? Just so users are now aware the format of the date will change on the files.

Reply:
But the default stays the same, no?

Reply:
Yes, the $timeNow is used for naming the new parent folder that gets created each time the export is done. It will use the default provided by formatting.uformat (yyyyMMddHHmmss as requested in #6837).

I will add a note regarding this in the command description.